src/defaults/html.pipe.js

```diff
@@ -77,7 +77,7 @@ const htmlpipe = (cont, context, action) => {
     .use(cache).when(uncached)
     .use(key)
     .use(tovdom).expose('post') // start HTML post-processing
-    .use(removeHlxProps).when(() => production())
+    .use(removeHlxProps).expose('removeHlxProps')
```

Comment:
I'd prefer a lowercase extension point name for consistency sake.

```suggestion
    .use(removeHlxProps).expose('cleanup')
```

Reply:
Is there a "debug" mode in the pipeline? If yes, would it make sense to leave them in that case?